### PR TITLE
Add Mealie to Easypanel 🎉

### PR DESCRIPTION
**Pull request by Supernova3339**
This adds the popular open-source recipe book, Mealie to Easypanel!
Tested successfully on `ARM` and `AMD` CPUs.

### CODE REVIEW

File: templates/mealie/index.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+export function generate(input: Input): Output {
+  const services: Services = [];
+
+  services.push({
+    type: "app",
+    data: {
+      projectName: input.projectName,
+      serviceName: input.appServiceName,
+      env: [
+        `DEFAULT_EMAIL=changeme@easypanel.io`,
```
Comment (deiucanta) [inline templates/mealie/index.ts:13]: If this is required, please extract it in a separate field

File: templates/mealie/index.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          name: "appdata",
+          mountPath: "/app/data",
+        },
+        {
+          type: "volume",
+          name: "mealie",
+          mountPath: "./mealie/data",
+        },
+      ],
+            domains: [
```
Comment (deiucanta) [inline templates/mealie/index.ts:37]: This doesn't seem to be properly formatted.

**Review by deiucanta — CHANGES_REQUESTED** [review]
Also, make sure you remove the changes in `package-lock.json` and `yarn.lock`. No need to update those for a template

**Comment by Supernova3339** [comment]
> Also, make sure you remove the changes in `package-lock.json` and `yarn.lock`. No need to update those for a template

I always run npm I, will make sure to not PR those as well

**Comment by deiucanta** [comment]
Merged manually

**Pull request closed without merging**